src/Layer/FeatureGeometryLayer.js

```diff
@@ -0,0 +1,44 @@
+import { Group } from 'three';
+
+import GeometryLayer from 'Layer/GeometryLayer';
+import FeatureProcessing from 'Process/FeatureProcessing';
+import Feature2Mesh from 'Converter/Feature2Mesh';
+
+/**
+ * `FeatureGeometryLayer` displays geographic vector data (geojson, kml...) in object 3D.
+ * `FeatureGeometryLayer` is a pre-configured `GeometryLayer` to load and convert vector data.
+ * In effect, `GeometryLayer` allows to customize the loading of data (`update` method)
```

Comment:
Could you replace by the following ?

_In deed, `GeometryLayer` allows customizing data loading (`update` method)..._

Reply:
ok